File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/event/exception/CannotOpenEventException.java
package band.gosrock.domain.domains.event.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class CannotOpenEventException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new CannotOpenEventException();

    private CannotOpenEventException() {
        super(EventErrorCode.CANNOT_OPEN_EVENT);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/event/exception/CannotOpenEventException.kt
package band.gosrock.domain.domains.event.exception

import band.gosrock.common.exception.DuDoongCodeException

class CannotOpenEventException private constructor() : DuDoongCodeException(EventErrorCode.CANNOT_OPEN_EVENT) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = CannotOpenEventException()
    }
}
